src/Microsoft.Azure.SignalR.Management/HttpRequest.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    internal class HttpRequest
```

Comment:
>HttpRequest [](start = 19, length = 11)

what this class for?

Reply:
The naming is too general

---
In reply to: [245872032](https://github.com/Azure/azure-signalr/pull/323#discussion_r245872032) [](ancestors = 245872032)

Reply:
For call rest api

Reply:
> The naming is too general
> 
> In reply to: [245872032](https://github.com/Azure/azure-signalr/pull/323#discussion_r245872032) [](ancestors = 245872032)

How about `RestApiRequest`

Reply:
`...Helper` or `...Utility`?

Reply:
OK. sounds `..Helper` / `..Utility` is more usual